src/pineko/cli/fonll.py

```diff
@@ -0,0 +1,163 @@
+"""CLI entry point to FONLL."""
+import pathlib
+
+import click
+import rich
+
+from .. import configs, fonll, parser, theory_card
+from ._base import command
+
+config_setting = click.option(
+    "-c",
+    "--configs",
+    "cfg",
+    default=None,
+    type=click.Path(resolve_path=True, path_type=pathlib.Path),
+    help="Explicitly specify config file (it has to be a valid TOML file).",
+)
+
+
+class TheoryCardError(Exception):
+    """Raised when asked for FONLL theory cards with an original tcard as input that is not asking for FONLL."""
+
+    pass
```

Comment:
```suggestion
```

Stupid remark: you do not need `pass`, it's enough to have a docstring :)
(there has to be something in, and a docstring is something)